solidity/nft_token.sol

```diff
@@ -10,42 +10,43 @@ struct Position {
     int32 tick_upper_index; // 4
 }
 
-@program_id("2cptovuGx5eyxkbdC6g3C1m3a4W7gJ7KemjBBB2Cthx8")
+@program_id("GUGGHzwC8wEKY3g7QS38YmoS8t5Q2faWAGAfxDK2bXbb")
 contract nft_token {
-    address public orca;
-    address public whirlpool;
+    address public constant orca = address"whirLbMiicVdio4qvUfM5KAg6Ct8VwpYzGff3uctyCc";
+    address public pool;
     address public pdaProgram;
-    address public mintErc20;
-    address public pdaERC20Account;
-    // TODO: Change for the latter one in production
-    uint64 public pdaHeader = 0x55bdfe33;//0xd0f7407ae48fbcaa;
-    // PDA seed
+    address public bridgedTokenMint;
```

Comment:
why we are using the word "bridged" here?

Reply:
This is the mint account of the bridged token - the token that will be bridged.